### ISSUE TITLE
pasting a media link without doing embedding

### ISSUE THREAD

**Issue by modernNeo** [arturolinares/sn-rme#29]
If I paste a video link on the web-app in rich markdown editor, it refuses to not embed the video. I have to switch back to plain text editor, paste the link and then switch to rich markdown editor

**Comment by arturolinares** [comment]
I can't reproduce this. Tested in Firefox too. Do you have more details (like the browser, platform, adblocks...)

**Comment by modernNeo** [comment]
sure, just go to the standardnotes app. switch to rich markdown editor and then try to paste  https://www.youtube.com/watch?v=wp0sEXWSMwk without having it embed.

**Comment by mrtpcet** [comment]
Hello, I would like to not embed a youtube link to, because that's brokes my organisation's note. It is a way to not automatically embed a youtube or any other link ?
Thanks !

**Comment by N-Prime** [comment]
To paste a YouTube link already on the clipboard without embedding press Ctrl-k to open a dialogue box to "search or paste a link" then Ctrl-v to paste and then Enter.

**Comment by modernNeo** [comment]
> To paste a YouTube link already on the clipboard without embedding press Ctrl-k to open a dialogue box to "search or paste a link" then Ctrl-v to paste and then Enter.

Is it possible to change that so that all someone needs to do is Ctrl+Shift+V? That is already a well establish way to do `Paste without formatting" and I feel like that is applicable here.

**Comment by N-Prime** [comment]
Yeah, I noticed that a few weeks after I made the comment (sorry!).

The workaround I currently use is to Ctrl+k, paste the youtube link, and delete "https://" from the start. That keeps it from changing to an embed after clicking back to the note.

**Comment by vtwang** [comment]
I too would like the option to turn embedding for youtube off
